src/components/KTable/ColumnVisibilityMenu.vue

```diff
@@ -68,13 +99,15 @@
 </template>
 
 <script setup lang="ts">
-import { ref, watch, onBeforeMount, onMounted, type PropType, nextTick } from 'vue'
+import { ref, watch, onBeforeMount, onMounted, type PropType, nextTick, computed } from 'vue'
```

Comment:
we typically break out the type imports into a separate import
```suggestion
import { ref, watch, onBeforeMount, onMounted, nextTick, computed } from 'vue'
import type { PropType } from 'vue'
```